### ISSUE TITLE
Input Signal Flow Problems

### ISSUE THREAD

**Issue by georgkrause** [openAVproductions/openAV-Luppp#228]
in `looper.cxx` line 121-123:

```
			float* inputL = buffers->audio[Buffers::MASTER_INPUT_L];
			float* inputR = buffers->audio[Buffers::MASTER_INPUT_R];
			clips[clip]->record( nframes, inputL, inputR);
```

Which means that the input is directly recorded and the input volume fader does not have any effect on the volume of the clips. So this fader is basically useless, because you can control the volume of the input passed to the outputs with another knob. We could remove the fader or record the clips after the fader.

**Comment by georgkrause** [comment]
Actually, it was wrong what i said. The input volume effects other outs as well, but not the Clip Recorder. So we should change this. 

But there is another bug in the signal chain from my point of view:
When the Mix Button is turned off, but the send is turned on, you have output on the sends but not on the master out. So you eg have reverb on the input signal without hearing the input signal at all. Would make more sense for me to turn the effect send off if the monitoring is turned off.

**Comment by georgkrause** [comment]
fixed because of #230
